.github/workflows/links.yml

```diff
@@ -0,0 +1,26 @@
+name: Links
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *" # Every day at midnight
+  pull_request:
+    paths:
+      - ".github/workflows/links.yml"
```

Comment:
Can we also run the job, whenever an `.md` file is edited, to make sure PRs dont introduce any broken links accidentally.

Reply:
In case of existing links are broken, this will block new PRs.
I'm not sure if we should be on the critical path here, blocking existing workflows, and/or running these checks on every PR.

Reply:
This PR already fixes all the broken links right. So in that case, if we ignore the `releases/` directory; Ideally there should not be any broken links existing in the repo.